storage/watchable_store_test.go

```diff
@@ -15,47 +15,216 @@
 package storage
 
 import (
+	"bytes"
+	"fmt"
 	"os"
+	"strings"
 	"testing"
+
+	"github.com/coreos/etcd/storage/storagepb"
+)
+
+var (
+	testN      = 100
+	testKey    = []byte("Foo")
+	testValue  = []byte("Bar")
+	testKeys   = make([][]byte, testN)
+	testValues = make([][]byte, testN)
 )
 
-func TestWatch(t *testing.T) {
```

Comment:
Sure. Will add only a test for the new function.

Sincerely,
Gyu-Ho Lee
On Dec 31, 2015 13:54, "Xiang Li" <notifications@github.com> wrote:

> In storage/watchable_store_test.go
> <https://github.com/coreos/etcd/pull/4091#discussion_r48672214>:
>
> >  )
> >
> > -func TestWatch(t *testing.T) {
>
> Can we keep pervious test unchanged in the pull request? We just need to
> add a test for newWatchingToEventMap.
>
> If we want to refactor the test, we should do it in a separate pull
> request.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/coreos/etcd/pull/4091/files#r48672214>.
>